content/docs/user-guide/build/troubleshooting.md

```diff
@@ -50,6 +50,38 @@ These values are stored in the CMake cache. Clean the cache and reconfigure by d
 
 After cleaning the cache, the correct compiler should be detected during the CMake configuration stage.
 
+## CMake fails to find `Findo3de.cmake` 
+
+**Issue:** The CMake tool reports it cannot find a package configuration file provided by "o3de". This produces a warning similar to:
+
+```cmd
+CMake Error at CMakeLists.txt:10 (find_package): 
+
+By not providing "Findo3de.cmake" in CMAKE_MODULE_PATH this project has 
+asked CMake to find a package configuration file provided by "o3de", but 
+CMake did not find one. 
+
+Could not find a package configuration file provided by "o3de" with any of 
+the following names: 
+
+o3deConfig.cmake 
+
+o3de-config.cmake
+
+```
+
+**Remedy:** This issue is usually caused by a misconfigured `o3de_manifest.json` file that has an entry for a version of O3DE that no longer exists or has missing or corrupt files.  This can occur if you installed O3DE and then deleted the engine folder instead of uninstalling it using "Add or Remove Programs".  Do the following:
+1. Close `o3de.exe` if it is open.
```

Comment:
Clarify which application this is, for people who might not know that `o3de.exe` is Project Manager.
```suggestion
1. Close Project Manager (`o3de.exe`) if it is open.
```